src/screens/LiquidityPools/LiquidityPools.js

```diff
@@ -288,7 +288,8 @@ const LiquidityPoolsScreen = ({
 
   const renderPurchasedPool = ({ item: pool, index }) => {
     const poolStats = poolsStats[index];
-    const balance = poolStats.userLiquidityTokenBalance;
+    const balance = poolStats.userLiquidityTokenBalance.toNumber();
```

Comment:
`toNumber`?

Reply:
This is only a rough calculation, no harm to be done here.